action.yml

```
@@ -65,6 +65,11 @@ inputs:
     description: >-
       Directory that contains actions-runner software and scripts. E.g. /home/runner/actions-runner.
     required: false
+  pre-runner-script:
+    description: >-
+      Bash commands to run before the actions-runner runs.  This is useful for installing dependencies.
```

Let's use similar description to the one from the README:

```suggestion
      Specifies bash commands to run before the runner starts. It's useful for installing dependencies with apt-get, yum, dnf, etc.
```